langchain4j-mcp/src/main/java/dev/langchain4j/mcp/client/ToolExecutionHelper.java

```diff
@@ -0,0 +1,25 @@
+package dev.langchain4j.mcp.client;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+
+public class ToolExecutionHelper {
```

Comment:
```suggestion
class ToolExecutionHelper {
```

Or maybe put it in the `internal` sub package if you want to reuse it from Quarkus?

Reply:
Right now I don't think Quarkus will need it, so ok I'll make it package-private